lab8/8-4.js

```diff
@@ -0,0 +1,50 @@
+(function() {    
+    var input = prompt('input your card number'),
+        card = parseInt(input),
+        nDigits = Math.log10(card) + 1 | 0, //valid for positive integers
+        cardType = 'INVALID';
+   
+    if (nDigits == 15 && (card >= 340000000000000 || card >= 370000000000000)) {
```

Comment:
This will also be true for 35xxxxxxxxxxxxx, 36xxxxxxxxxxxxx, 38xxxxxxxxxxxxx, and whatever bigger than 340000000000000. I don't think it's the right check.

Reply:
the purpose of this check is to ensure that card number begins from 35 || 36 || 38... 
it is stated that input date is positive inreger

Reply:
I believe I fixed it - see comment below